### PR TITLE
Popup menu

### PR DESCRIPTION
**Pull request by sibteali786**
## Describe your changes
Added a PopMenu Widget
## Issue ticket number and link
#67 
## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [ ] I have added tests

### CODE REVIEW

File: packages/runtime/src/widgets/PopupMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  const { values } = useRegisterBindings(props, props.id);
+  const onTapCallback = useExecuteCode(onItemSelect, values);
+  const onNavigate = useNavigateScreen(props.onItemSelect?.navigateScreen);
+  const [widgetProps, setWidgetProps] = useState<EnsembleWidget>();
+  const menuItems = (
+    <Menu>
+      {props.items.map((item) => (
+        <Menu.Item
+          key={item.value}
+          onClick={onItemSelect ? onTapCallback : onNavigate}
```
Comment (evshi) [inline packages/runtime/src/widgets/PopupMenu.tsx:56]: You will need to pass in the value to the callback so you know which item was selected

File: packages/runtime/src/widgets/PopupMenu.tsx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+          ...mergedStyles,
+        }}
+      />
+    </AntdDropdown>
+  );
+};
+
+WidgetRegistry.register("PopupMenu", PopupMenu);
+
+function renderWidget(widget: EnsembleWidget): ReactNode {
```
Comment (evshi) [inline packages/runtime/src/widgets/PopupMenu.tsx:100]: You can just use EnsembleRuntime.render?

File: packages/runtime/src/widgets/PopupMenu.tsx (hunk 1 above)
Comment (evshi) [reply packages/runtime/src/widgets/PopupMenu.tsx:56]: I'll have an update to the actions hook that will make this easier.

File: packages/runtime/src/widgets/PopupMenu.tsx (hunk 2 above)
Comment (sibteali786) [reply packages/runtime/src/widgets/PopupMenu.tsx:100]: sure, will refactor that

**Comment by evshi** [comment]
Closing since this was merged with demo branch

**Pull request closed without merging**